<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProfile</class>
 <widget class="QWidget" name="FormProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="LabelAvatar">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">Avatar</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="LabelNick">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">NickName</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonSetProfile">
           <property name="text">
            <string>Это мой профиль</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="LabelPersonaState">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ebrima</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">PersonaState</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>108</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonUpdate">
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="LabelProfileUrl">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ebrima</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">ProfileUrl</string>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextBrowserInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="Labellvl">
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">lvl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelTimeCreated">
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">TimeCreated</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelLocCountryCode">
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">CountryCode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelRealName">
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">RealName</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="LabelProfileVisibility">
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Visibility</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelBans">
       <property name="font">
        <font>
         <family>Ebrima</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Bans</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="ButtonGames">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">Games</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonFriends">
       <property name="text">
        <string notr="true">Friends</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonStatistics">
       <property name="text">
        <string> Статистика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonFavorites">
       <property name="text">
        <string> Избранное</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonGoToMyProfile">
       <property name="text">
        <string>На мой профиль</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
